<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="horizontalWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="QRadioButton" name="rb_encryption">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Encryption</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QRadioButton" name="rb_decryption">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Decryption</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="mainfield">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>7</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="label_OUT">
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Output</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_IN">
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Input</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="field_output">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>180</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans Light</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="field_input">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>180</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans Light</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="pb_cryption">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
       </font>
      </property>
      <property name="text">
       <string>Cryption</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionCryption">
   <property name="text">
    <string>Cryption</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionCryption</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>cryption()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_cryption</sender>
   <signal>clicked()</signal>
   <receiver>actionCryption</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cryption()</slot>
  <slot>change_size()</slot>
 </slots>
</ui>
